contents/docs/surveys/implementing-custom-surveys.mdx

```diff
@@ -202,33 +203,58 @@ export default function Feedback() {
 
 ## Capture survey responses
 
-The main event you must capture to track survey results is `survey_sent` which we've already shown. It is formatted like this:
+The main event you must capture to track survey results is `survey sent`. PostHog supports two formats for survey responses, but we strongly recommend using ID-based responses as they are more reliable and resistant to changes in question ordering:
 
 ```js-web
 posthog.capture("survey sent", {
     $survey_id: survey.id, // required
-    $survey_response: survey_response // required. `survey_response` must be a text value.
+    $survey_response_question_id: survey_response // ID-based (recommended)
 })
 ```
 
+For backward compatibility, PostHog also supports index-based responses, but these are more fragile as they depend on question order:
+
+```js-web
+posthog.capture("survey sent", {
+    $survey_id: survey.id,
+    $survey_response: survey_response // Index-based (legacy, position 0)
+})
+```
+
+### Response formats
+
 Survey responses expect text, so you should convert numbers to text e.g. `8` should be converted to `"8"`.
 
-For multiple choice surveys, `survey_response` must be an array of values with the selected choices e.g., `$survey_response: ["response_1", "response_2"]`.
+For multiple choice surveys, the response must be an array of values with the selected choices e.g., `$survey_response_multiple_choice_1: ["response_1", "response_2"]`.
 
 ### Capturing multiple responses
 
-If you have a survey with multiple questions, you can capture the responses to each question using the following syntax:
+For surveys with multiple questions, use ID-based properties to ensure your responses remain valid even if questions are reordered:
 
 ```js-web
 posthog.capture("survey sent", {
     $survey_id: survey.id, // required
-    $survey_response: survey_response,
-    $survey_response_1: survey_response_1,
-    $survey_response_2: survey_response_2
+    
+    // Using question IDs (recommended)
+    $survey_response_open_text_1: first_question_response,
+    $survey_response_nps_rating_1: second_question_response,
+    $survey_response_multiple_choice_1: third_question_response
+})
+```
+
+While index-based responses are still supported, we recommend using ID-based responses for all new implementations:
+
+```js-web
+// Legacy format (not recommended for new implementations)
+posthog.capture("survey sent", {
+    $survey_id: survey.id,
+    $survey_response: first_question_response,
+    $survey_response_1: second_question_response,
+    $survey_response_2: third_question_response
```

Comment:
this is a good example because it reflects a real example, differently than eg `$survey_response_open_text_1`